src/Whoops/Handler/PrettyPageHandler.php

```diff
@@ -137,6 +142,19 @@ public function handle()
             $code = Misc::translateErrorCode($inspector->getException()->getSeverity());
         }
 
+        // Detect frames that belong to the application.
+        if ($this->applicationPaths) {
+            /* @var \Whoops\Exception\Frame $frame */
+            foreach ($frames as $frame) {
+                foreach ($this->applicationPaths as $path) {
+                    if (substr($frame->getFile(), 0, strlen($path)) === $path) {
```

Comment:
It seems slightly more practical to do paths. You usually know your application code is in `whatever/src`. And then whether its simple functions or plain files, incorrect namespaces, it will all work here.